### PR TITLE
Prepare 1.11.0

### PR DESCRIPTION
**Pull request by leplatrem**
**Step 1**
```
$ git co -b prepare-X.X
$ prerelease
$ vim docs/conf.py
$ git ci -a --amend
$ git push origin prepare-X.X
```

* [ ] Merge remaining pull requests
* [x] Update changelog
* [x] Update version in docs/conf.py
* [x] Version dependencies
* [x] If cliquet was updated, update the link to default settings in the docs
* [x] if cliquet protocol was updated, upgrade API protocol in docs/

**Step 2**
```
$ git co master
$ git merge --no-ff prepare-X.X
$ release
$ postrelease
```
* [ ] Tag vX.Y.Z
* [ ] Publish on Pypi
* [ ] Revert version pinning + revert master url in dev-requirements.txt

**Step 3**
* [ ] Close milestone in Github
* [ ] Add entry in Github release page
* [ ] Create next milestone in Github
* [ ] Configure the version in ReadTheDocs
* [ ] Send mail to ML
* [ ] Tweet!

### CODE REVIEW

**Comment by leplatrem** [comment]
@Natim @n1k0 please review the changelog with care r?

**Comment by n1k0** [comment]
LGTM

**Comment by Natim** [comment]
LGTM too.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,23 +3,83 @@ Changelog
...
+- The API capabilities can be exposed in a ``capabilities`` attribute in the
+  root URL (#628). Clients can rely on this to detect optional features on the
+  server (e.g. enabled plugins)
+
+Protocol is now version 1.3. See `API changelog <http://kinto.readthedocs.org/en/latest/api/>`_.
+
+**New features**
+
+- Add a Heroku single-clic deploy button (#362)
+- Install postgresql libraries on kinto init (fixes #313)
```
Comment (Natim) [inline CHANGELOG.rst]: nit: PostgreSQL

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,23 +3,83 @@ Changelog
...
+
+Minor:
 
 - Added missing DELETE endoint for list of records (fixes #238)
 - Mention how to restrict private URLs with NGinx (fixes #250)
-- Do not require cliquet master branch in dev. Now moved as tox env in TravisCI
+- Fix link to the freenode #kinto channel in the docs (#333)
+- Remove Firefox Account mention from README (fixes #326)
+- Move application examples page to wiki (ref #321)
+- Move postgresql server docs to wiki (fixes #321)
```
Comment (Natim) [inline CHANGELOG.rst]: PostgreSQL

File: CONTRIBUTORS.rst [hunk 3]
```diff
@@ -2,17 +2,23 @@ Contributors
...
+* Nicolas Hoizey <nicolas@hoizey.com>
 * Niraj <https://github.com/niraj8>
-* Rémy Hubscher <rhubscher@mozilla.com>
+* rektide <rektide@voodoowarez.com>
+* Rémy HUBSCHER <rhubscher@mozilla.com>
 * Shweta Oak <oakshweta11@gmail.com>
 * Sofia Utsch <sofia.utsch@gmail.com>
-* Tarek Ziade <tarek@mozilla.com>
-* John Giannelos <jgiannelos@mozilla.com>
+* Tarek Ziade <tarek@ziade.org>
```
Comment (Natim) [inline CONTRIBUTORS.rst]: Why change to ziade.org here?

File: CONTRIBUTORS.rst [hunk 4]
```diff
@@ -2,17 +2,23 @@ Contributors
 ============
 
 * Aaron Egaas <me@aaronegaas.com>
-* Alexis Metaireau <alexis@mozilla.com>
+* Aditya Bhasin <conlini@gmail.com>
+* Alexis Métaireau <alexis@notmyidea.org>
```
Comment (Natim) [inline CONTRIBUTORS.rst]: Same question here, why change to notmyidea.org?

File: setup.py [hunk 5]
```diff
@@ -28,11 +28,11 @@ def read_file(filename):
...
-    'cliquet[postgresql]>=2.14,<3'
+    'cliquet[postgresql]>=2.15,<3'
 ]
 
 MONITORING_REQUIREMENTS = REQUIREMENTS + [
-    'cliquet[monitoring]>=2.14,<3'
+    'cliquet[monitoring]>=2.15,<3'
 ]
 
 FXA_REQUIREMENTS = REQUIREMENTS + [
```
Comment (Natim) [inline setup.py]: We should probably tag the cliquet-fxa version as well.

File: CONTRIBUTORS.rst (hunk 3 above)
Comment (leplatrem) [inline CONTRIBUTORS.rst]: that's what git log gave me. I'll put it back

**Pull request merged**